plugins/catalog-backend/src/ingestion/types.ts

```diff
@@ -79,7 +79,7 @@ export type EntityMeta = {
    * Key/value pairs of non-identifying auxiliary information attached to the
    * entity.
    */
-  annotations?: Record<string, string>;
+  annotations?: Record<string, string | undefined>;
```

Comment:
this is actually not right as far as i know - these must always be strings

Reply:
- [x] remove `| undefined`